### PR TITLE
fix: check box issue

### LINKED ISSUE #106: [Bug]: check-box not working properly
### Describe the bug

After creating multiple forms at dashboard when we check a particular form using check-box it will check all the available form

### To Reproduce

1. Sign up or log in 
2. Go to 'http://app.localhost:3000/dashboard/'
3. Create multiple forms
4. Click on check-box

### Expected behavior

Check only one element at a time

### Screenshots/Videos

https://github.com/piyushgarg-dev/review-app/assets/133745159/f84ba216-0853-43ab-bab5-e161d0cab3f6

### Additional context

_No response_

### Please checkmark the following checklist

- [X] I make sure that similar issue is not opened

### PR DESCRIPTION
**Pull request by TusharKumawat1**
## What does this PR do?

In this pr i fix the issue of selecting all available check-box at a time 

Fixes #106

https://github.com/piyushgarg-dev/review-app/assets/133745159/1484ddf6-d341-41f0-9e4e-3c1bce8a9cbc

## Type of change
bug fix

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Mandatory Tasks

- [ x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Review by hemantwasthere — CHANGES_REQUESTED** [review]

File: gql/graphql.ts [hunk 1]
```diff
@@ -73,6 +73,28 @@ export type Form = {
   updatedAt?: Maybe<Scalars['Date']['output']>;
```
[the remark this answers is not in the document]
Comment (TusharKumawat1) [reply gql/graphql.ts:73]: Hi, I'm new to open source, and I have doubts about how to discard file changes. Can you guide me?

[the remark this answers is not in the document]
Comment (TusharKumawat1) [reply gql/graphql.ts:73]: Hi, I'm new to open source, and I have doubts about how to discard file changes. Can you guide me?

File: components/Dashboard/FormPane.tsx [hunk 2]
```diff
@@ -47,10 +47,16 @@ const FormPane: React.FC = () => {
   const router = useRouter()
 
   const [selectedRow, setSelectedRow] = useState(false)
-
+  // Initialize an array to hold the state of each checkbox
+  const [selectedRows, setSelectedRows] = useState([])
+  const handleCheckboxChange = (index) => {
```
Comment (piyushgarg-dev) [inline components/Dashboard/FormPane.tsx:52]: Please wrap this in `useCallback` hook

**Review by piyushgarg-dev — CHANGES_REQUESTED** [review]

**Comment by piyushgarg-dev** [comment]
Great One 🎉

File: components/Dashboard/FormPane.tsx (hunk 2 above)
Comment (TusharKumawat1) [reply components/Dashboard/FormPane.tsx:52]: Could you please explain to me how to commit changes in a pull request?
I made an effort, but it appears that my approach was incorrect.

File: gql/graphql.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (hemantwasthere) [reply gql/graphql.ts:73]: Just go to the previous commit of this commit and copy-paste the content of the files that you want to discard

**Comment by Mosam219** [comment]
@TusharKumawat1 i have checked your pr one issue is there. you are differentiating forms with index. But in future if we add more actions after selecting forms we need form Id so that issue i have solved in my pr you can check that.

cc: @piyushgarg-dev

**Pull request closed without merging**